uClibc build fix: program_invocation_short_name is defined in errno.h
and not argp.h
Update to util-linux-2.22.2 from a previous patch by Khem.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura util-linux-2.22.2.orig/configure.ac util-linux-2.22.2/configure.ac
--- util-linux-2.22.2.orig/configure.ac	2013-03-07 15:16:18.912418476 -0300
+++ util-linux-2.22.2/configure.ac	2013-03-07 15:16:25.909640745 -0300
@@ -372,7 +372,7 @@


 AC_MSG_CHECKING(whether program_invocation_short_name is defined)
-AC_TRY_COMPILE([#include <argp.h>],
+AC_TRY_COMPILE([#include <errno.h>],
	       [program_invocation_short_name = "test";],
	       AC_DEFINE(HAVE_PROGRAM_INVOCATION_SHORT_NAME, 1,
			 [Define if program_invocation_short_name is defined])
